Podfile

```diff
@@ -5,7 +5,7 @@ target 'NthKey' do
   # Comment the next line if you don't want to use dynamic frameworks
   use_frameworks!
 
-  pod 'LibWally', :git => 'https://github.com/sjors/LibWally-Swift.git', :tag => "v0.0.6", :submodules => true
+  pod 'LibWally', :git => 'https://github.com/jurvis/libwally-swift.git', :branch => "jurvis/build-from-xcode", :submodules => true
```

Comment:
try with  

`pod 'LibWally', :git => "https://github.com/jurvis/libwally-swift.git", :branch => "jurvis/cleanup-build-script-2", :submodules => true`

instead -- `jurvis/build-from-xcode` is my "dirty" branch with unsquashed commits so I could debug CI lol

Reply:
Still getting the same error though: https://termbin.com/x28m

Reply:
hm, do you have the same problem with the [sample project](https://github.com/jurvis/LibwallyCocoapodsSample)?

Reply:
Yes (I also had to remove `Podfile.lock`)

Reply:
`$NATIVE_ARCH` isn't set for me in a regular shell anyway.

Reply:
What version of Ruby are you using?

Reply:
I just tried with https://github.com/Sjors/libwally-swift/pull/69/commits/f3ca7a040feefecd080df732312d70759f2a010f but no luck.

Reply:
Neither did https://github.com/Sjors/libwally-swift/pull/69/commits/fc2e4f3992eb746bc70dfd0c9c73a49d9d24dc2d

(gotta go now, will test again tomorrow)

Reply:
bummer, thanks for testing! I will keep trying and squash once I figure it out :)